src/pages/core/guides/oracle-plugin-example.md

```diff
@@ -0,0 +1,375 @@
+
+### Oracle Plugin
+
+The **Oracle Plugin** leverages the capability of external plugins to save data that an external authority can update by accessing **on-chain data** accounts external to the Core asset, allowing assets to dynamically reject lifecycle events set by the asset authority. The external Oracle account can also be updated at any time to change the authorization behavior of the lifecycle events, providing a flexible and dynamic experience.
+
+If you want to learn more about the Oracle Plugin, read more about it [here](/core/external-plugins/oracle).
+
+## Starting off: Understanding the Protocol behind the Idea
+
+To create an NFT collection that can only be traded during US market hours, we need a reliable way of updating on-chain data based on the time of day. This is how the protocol design will look like:
+
+### Program Overview
+
+The program will have two main instructions (one to create the Oracle and the other to update its value) and two helper functions to facilitate implementation.
+
+**Main Instructions**
+- **Initialize Oracle Instruction**: This instruction creates the oracle account so any user wanting to employ this time-gated feature for their collection will redirect the NFT Oracle Plugin to this on-chain account address.
+- **Crank Oracle Instruction**: This instruction updates the oracle state data to ensure it always has the right and most up-to-date data.
+
+**Helper functions**
+- **isUsMarketOpen**: Checks if the US market is open.
+- **isWithin15mOfMarketOpenOrClose**: Checks if the current time is within 15 minutes of market opening or closing.
+
+**Note**: The `crank_oracle_instruction` ensure that the protocol is updated with accurate data providing incentives to those maintaining up-to-date information. But we'll talk about this in the next section.
+
+### The Incentives mechanism
+
+Every collection using this oracle as a source of trust should run its own crank to ensure that the oracle is always up-to-date. However, to enhance resilience, protocol developers should consider creating incentives for multiple people to crank the protocol, ensuring a safety net that keeps the oracle data accurate if the in-house crank fails to update the data.
+
+The current program design rewards crankers for maintaining the oracle with 0.001 SOL. This amount is manageable while still providing a sufficient incentive for crankers to keep the oracle state account up-to-date.
+
+Note: These incentives are paid out only if the crank is executed within the first 15 minutes of market opening or closing and are funded from a vault present in the smart contract. The vault needs to be refilled by sending SOL to the oracle vault address.
+
+**Note**: These incetives are paid out only if the crank is executed the first 15 minute of market opening or closing and are funded from a vault present in the smart contract. The vault needs to be refilled by sending SOL to the oracle vault address.
```

Comment:
there is a duplicate here.
We also have different types of callouts that you could use to make the note more visible

Reply:
What type of callouts would work best here in your opinion?

Reply:
Probably 
`{% callout type="note" %}`